src/Rules/Resources/HowToFix.resx

```diff
@@ -142,68 +145,68 @@
+    <value>Write meaningful text for the element's UI Automation HelpText property. This text is typically similar to, or taken from, a tooltip or placeholder, such as "type here to search".</value>
+    <comment>Brief guidance on how to fix an accessibility problem. The term "HelpText" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="IsContentElementFalseOptional" xml:space="preserve">
-    <value>The recommended value of the IsContentElement property for the given control type is false. Please consider if this is an element that should be reported to an assistive technology user as content.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "IsContentElement" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>If this element does not contain data that is presented to the user (such as items in a list box or a button in a modal dialog box), set the IsContentElement UI Automation property to false.</value>
+    <comment>Brief guidance on how to fix an accessibility problem. The term "IsContentElement" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="IsContentElementTrueOptional" xml:space="preserve">
-    <value>The recommended value of the IsContentElement property for the given control type is true. Please consider if this is an element that should be reported to an assistive technology user as content.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "IsContentElement" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>If this element contains data that is presented to the user (such as items in a list box or a button in a modal dialog box), set the IsContentElement UI Automation property to true.</value>
+    <comment>Brief guidance on how to fix an accessibility problem. The term "IsContentElement" should not be translated (i.e. it should be treated as a loanword).</comment>
   </data>
   <data name="IsControlElementTrueOptional" xml:space="preserve">
-    <value>The recommended value of the IsControlElement property for the given control type is true. Please consider if this is an element that should be reported to an assistive technology user as a control. Note that almost all controls are required to have the IsControl Property set to true.</value>
-    <comment>The description of a problem with an element currently under inspection. The term "IsControlElement" should not be translated (i.e. it should be treated as a loanword).</comment>
+    <value>If this element represents an interactive UI control, set the IsControlElement UI Automation property to true.</value>
```

Comment:
Looking at the rule, it might be simpler to just say something like "Set the IsControlElement UI Automation property for this element to true". The rule's condition should already filter out controls where the user doesn't need to change anything.

Reply:
Similar comment for IsControlElementFalseOptional, although it's not current in this PR